room/views.py

```diff
@@ -1,3 +1,100 @@
-from django.shortcuts import render
+from haversine        import haversine
+import json
 
-# Create your views here.
+from django.views     import View
+from django.http      import JsonResponse
+from django.db.models import Q
+
+from .models          import Room, Image, Agency, Status, FloorType, RoomType, StructureType, Parking, Direction
```

Comment:
```suggestion
from .models          import (
    Room,
    Image,
    Agency,
    Status,
    FloorType,
    RoomType,
    StructureType,
    Parking,
    Direction
)
```
이정도 개수는 이렇게 컨벤션 권장 드리고 있습니다.